README.md

```diff
@@ -2,6 +2,13 @@
 
 This is a repo for learning about how to use `git`.
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+By Lauren Tso
+=======
+By Kevin Bonham
+>>>>>>> b06cf0453e3a1c2c95bd6383ca38a28b905625f2
+=======
```

Comment:
This is caused by a merge conflict. Probably best to just discard chances to this file (you can do this by doing:

```
$ git reset --hard README.md
```